src/messages/messages.ts

```diff
@@ -106,6 +107,10 @@ export const issuanceMessage = {
   fetchingIssuances: 'Fetching project issuances...',
   selectIssuance: 'Select an Issuance',
   noIssuances: "You don't have any issuances to choose from",
+  vcBeingVerified: 'VC verification in progress...',
+  vcVerified: 'VC Verified',
+  vcNotVerified: 'VC Not Verified',
```

Comment:
I think "VC is valid" / "VC is not valid" are better, because in both cases it has been verified, but those are semantics, not very important